packages/core/src/components/info/info.global.scss

```diff
@@ -1,7 +1,5 @@
-@use "~@dso-toolkit/sources/src/components/info/info";
-
 dso-info {
-  .dso-rich-content[slot="info"] {
-    @include info.info_richContent();
+  .dso-rich-content {
+    @include info_richContent();
```

Comment:
Nog een keer naar deze mixin kijken: Kan deze `_children()` worden en kan de `.dso-rich-content` selector erin? Vergeet niet dat deze mixin ook in selectable.scss wordt aangeroepen

Reply:
Selectable had ook de `[slot="info"]` maar aangezien je die zelf ook al weg hebt gehaald in `info.global.scss` ben ik zo vrij geweest om hem ook bij `selectable` weg te halen. Dan kan de mixin inderdaad `.dso-rich-content` hebben en hernoemt worden naar `_children`

Reply:
Alweer teruggedraaid, ziet er beter uit zo